Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.10
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "TruexAdRenderer-iOS",
+    products: [
+        // Products define the executables and libraries a package produces, making them visible to other packages.
+        .library(
+            name: "TruexAdRenderer-iOS",
+            targets: ["TruexAdRenderer-iOS"]),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package, defining a module or a test suite.
+        // Targets can depend on other targets in this package and products from dependencies.
+        .binaryTarget(
+            name: "TruexAdRenderer-iOS",
+            url: "https://stash.truex.com/integration/TruexAdRenderer-iOS-v3.4.0.zip",
```

Comment:
is this the same one as for cocoa pods?
is it compatible?
i.e. we don't need an spm specific one?

Reply:
Yes this is the same build used for cocoapods, and it seems to be compatible with Swift Packages based on my own testing, since it doesn't actually have any dependencies, and thus there's no cocoapod presence to clash with a Swift Package workflow.